### PR TITLE
Optimize command examples in Lisk Docs - Closes 1827

### LINKED ISSUE #1827: Optimize command examples (General)
The description of the issue is described in the [epic](https://github.com/LiskHQ/lisk-docs/issues/1490).

### PR DESCRIPTION
**Pull request by TalhaMaliktz**
Closes #1827

### CODE REVIEW

File: docs/modules/ROOT/pages/build-blockchain/module/command.adoc [hunk 1]
```diff
@@ -32,13 +32,13 @@ On this page, you'll learn how to:
...
+====
 
 .Sample code
 [NOTE]
 =====
 View the complete sample code of this guide on GitHub in the {url_github_hello_command}[Lisk SDK examples repository^].
 =====
-====
 
 As defined on the xref:{url_guides_module}[] page, the xref:{url_understand_modules_commands}[command] shall provide the following functionality:
```
Comment (chris529) [inline docs/modules/ROOT/pages/build-blockchain/module/command.adoc]:
```suggestion
As defined on the xref:{url_guides_module}[] page, the xref:{url_understand_modules_commands}[command] shall provide the following functionalities:
```

File: docs/modules/ROOT/pages/integrate-blockchain/create-sign-transactions.adoc [hunk 2]
```diff
@@ -253,6 +253,7 @@ After entering all relevant information for the required transaction, the encode
 .Example output
 [%collapsible]
 ====
+[source,bash]
```
Comment (Tschakki) [inline docs/modules/ROOT/pages/integrate-blockchain/create-sign-transactions.adoc:256]:
```suggestion
[source,json]
```

Comment (Tschakki) [inline docs/modules/ROOT/pages/integrate-blockchain/create-sign-transactions.adoc]:
```suggestion
[source,json]
```

**Review by Tschakki — APPROVED** [review]
LGTM 🎉

**Review by chris529 — APPROVED** [review]
LGTM! I just left one minor suggestion. 👍

**Pull request merged**